app/src/main/java/com/github/mateo762/myapplication/profile/ProfileActivity.kt

```diff
@@ -117,18 +117,24 @@ class ProfileActivity : BaseActivity(), CoroutineScope {
         changeUsernameButton.visibility = View.GONE
         btnSave.visibility = View.GONE
 
-        uid = intent.getStringExtra("userId") ?: user!!.uid
+        uid = "mTFQAS8YmlXK89siWb36PwIe1x82"
```

Comment:
What is this fixed uid? If there's an exception it looks like it uses this one, but why?

Reply:
This is so when a test opens the profile activity, if it's not logged in it will take the id of a test user so the ProfileActivity is displayed correctly

Reply:
You are hacking the activity for a test, here is the correct way to do it: https://stackoverflow.com/questions/72154732/launching-activity-with-extra-attached-to-intent-in-instrumented-test-running-o